docs/remote/devcontainer-cli.md

```diff
@@ -37,6 +39,21 @@ To install the devcontainer CLI:
       -h, --help  Show help  [boolean]
     ```
 
+### Install devcontainer CLI outside VS Code
```

Comment:
@Chuxel Is this the area you're referring to in https://github.com/microsoft/vscode-docs/pull/4856#discussion_r722510301? i.e. we can update this section to be called "Install from the command line" and include the latest instructions on how to install the CLI?

But otherwise the content in the pre-building section makes sense?

Reply:
@bamurtaugh Yep! Christof also illustrated how to use it via npx.  So there's two ways to install and use the CLI:

Global install:
```
$ npm install -g https://github.com/microsoft/vscode-remote-release/releases/latest/download/dev-container-cli.tgz
$ devcontainer --help
```

Local install:
```
$ npm install https://github.com/microsoft/vscode-remote-release/releases/latest/download/dev-container-cli.tgz
$ npx @microsoft/dev-container-cli --help
```

Using npx causes it to use the package name.

To future proof a bit, we could aka.ms that URL and reference that instead.

Reply:
Thanks so much for this! Created https://aka.ms/dev-container-cli and updated this section.